tests/test_ipaddress_field_serializer.py

```diff
@@ -0,0 +1,97 @@
+from django.db import models
+from django.test import TestCase
+
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+
+# Define the model
+class TestModel(models.Model):
+    address = models.GenericIPAddressField(protocol="both")
+
+    class Meta:
+        app_label = "main"
+
+
+class TestSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = TestModel
+        fields = "__all__"
+
+
+# Define the serializer in setUp
+class TestSerializerTestCase(TestCase):
```

Comment:
I would have expected this test to be next to the existin ones:

https://github.com/encode/django-rest-framework/blob/28d0261afcd6702900512e00c37f4e264c117d83/tests/test_model_serializer.py#L411